exotica_python/src/Exotica.cpp

```diff
@@ -476,16 +496,6 @@ PYBIND11_MODULE(_pyexotica, module)
     py::class_<KinematicTree, std::shared_ptr<KinematicTree>> kinematicTree(kin, "KinematicTree");
     kinematicTree.def("publishFrames", &KinematicTree::publishFrames);
 
-    py::class_<KinematicFrameRequest, std::shared_ptr<KinematicFrameRequest>> kinFrameReq(kin, "KinematicFrameRequest");
```

Comment:
Wondering whether these should be removed? May be useful for debugging or when used in combination with other tools